test/functest/proto/xml__map__map.proto

```diff
@@ -43,26 +43,26 @@ message MapConf {
     protoconf.FruitType key = 1 [(tableau.field) = {name:"@key"}];
     protoconf.FruitType value = 2 [(tableau.field) = {name:"@value"}];
   }
-  map<string, Meal> meal_map = 9 [(tableau.field) = {name:"Meal" key:"Id"}];
+  map<string, Meal> meal = 9 [(tableau.field) = {name:"Meal" key:"Id"}];
   message Meal {
     string id = 1 [(tableau.field) = {name:"Id"}];
     google.protobuf.Duration time = 2 [(tableau.field) = {name:"Time" prop:{default:"30m"}}];
-    map<string, Meat> meat_map = 3 [(tableau.field) = {name:"Meat" key:"Name"}];
+    map<string, Meat> meat = 3 [(tableau.field) = {name:"Meat" key:"Name"}];
     message Meat {
       string name = 1 [(tableau.field) = {name:"Name"}];
       string weight = 2 [(tableau.field) = {name:"Weight"}];
     }
     VegetableInfo vegetable_info = 4 [(tableau.field) = {name:"VegetableInfo"}];
     message VegetableInfo {
-      map<string, Vegetable> vegetable_map = 1 [(tableau.field) = {name:"Vegetable" key:"Name" prop:{optional:true}}];
+      map<string, Vegetable> vegetable = 1 [(tableau.field) = {name:"Vegetable" key:"Name" prop:{optional:true}}];
```

Comment:
Should rollback the list and map field names for compatibility and easy use.

Reply:
done